File: conditionals/time_of_day.py
# James Wilcox, Time of Day Python

import time
# first instance of time
#print(time.gmtime(0))

#time in seconds since gmtime
current = time.time()

#current time as we are used to seeing it
now = time.ctime(current)
print(now)

#get just the hour
local_time = time.localtime(current)
hour = local_time.tm_hour
#print(hour)

if hour >= 6 and hour <= 11:
    print("Good Morning!")
elif hour >= 12 and hour <= 16:
    print("Good Afternoon!")
elif hour >=17 and hour <= 19:
    print("Good Evening")
else:
    print(f"You should be in bed!")

File: conditionals/time_of_day.c
// James Wilcox, Time of Day C
#include <stdio.h>
#include <time.h>

int main(void){
    time_t seconds;

    seconds = time(NULL);
    printf("Seconds since January 1, 1970 = %d\n", seconds);

    time_t rawtime;
    struct tm * timeinfo;

    time(&rawtime);
    timeinfo = localtime(&rawtime);
   //printf("Current time and date is %s\n", asctime(timeinfo));

    time_t now = time(NULL);
    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    printf("%d\n", hour);
    if (hour >= 6 && hour <= 11){
   printf("Good Morning!");
    }
   else if (hour >= 12 && hour <= 16){
   printf("Good Afternoon!");
   }
   else if (hour >= 17 && hour <= 19){
   printf("Good Evening!");
   }
   else{
    printf("You should be in bed!");
   }
    return 0;
}